data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+	variant 3
+		"Marauder Splinter (Weapons)"
+	variant 2
+		"Marauder Manta" 2
+	variant 1
+		"Marauder Leviathan (Engines)"
+	variant 1
+		"Marauder Raven" 2
+		
+
+mission "Unfettered: Nemo Me Impune Lacessit"
+	invisible
+	repeat
+	landing
+	minor
+	source
+		near "Ya Hai" 10
+	to offer
+		has "Unfettered: Jump Drive Retrieval: done"
+		not "sold bars"
+		random < 3
+	on offer
+		require "Gold Bars"
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 3
+		dialog `After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Maybe you possess something valuable to them? When they entered the system they only sent one message: "nemo me impune lacessit". Whatever it is they were up against, it is probably best to store it on a planet (using U) or get rid of it.`
+
+mission "Unfettered: Nemo Me Impune Lacessit 2"
+	invisible
+	repeat
+	landing
+	minor
+	source
+		near Sol 100
+	to offer
+		has "Unfettered: Jump Drive Retrieval: done"
+		not "sold bars"
+		random < 1
+	on offer
+		require "Gold Bars"
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 5
+		dialog `After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Maybe you possess something valuable to them? When they entered the system they only sent one message: "nemo me impune lacessit". Whatever it is they were up against, it is probably best to store it on a planet (using U) or get rid of it.`
+
+
+
+mission "Unfettered: I Shall Go Unpunished"
+	landing
+	source
+		government "Pirate"
+	to offer
+		random < 40
+		has "Unfettered: Jump Drive Retrieval: done"
+		has "flagship crew" > 4
+	on offer
+		require "Gold Bars" 5
+		conversation
+			`As you land on the pirate world, you head for the hatch, preparing to get on the surface, but passing by the cargo bay you think you hear a sound. Odd... There should not be anyone there right now, knowing your crew's habits.`
+			choice
+				`(Its nothing, probably just a rat).`
+					goto lostbars
+				`(Go investigate).`
+			`Arming yourself of your side weapon, you approach the bay as discretely as you can. Worst case scenario is you find one of your own crew, in which case you better not overeact, or someone could get hurt.`
+			choice
+				`(Ask if someone is there).`
+					goto hello
+				`(Never too careful).`
+			`You go into the cargo bay, on your guards. As you enter you think you saw something move, on your left. There is not much time to think.`
+			choice
+				`(Call my crew to help).`
+				`(Run in that direction).`
+					goto run
+			label help
+			`You call your crew to help and try to barricade the ship to prevent escape, but they cannot find anyone. You wonder how they could get away. Just to be sure, you check around the cargo bay. It seems like nothing is missing, but a few bars of golds are out of place. Maybe it was just a malfunction with the ship's gravitational systems.`
```

Comment:
```suggestion
			`You call your crew to help and try to barricade the ship to prevent escape, but they cannot find anyone. You wonder how they could get away. Just to be sure, you check around the cargo bay. It seems like nothing is missing, but a few bars of gold are out of place. Maybe it was just a malfunction with the ship's gravitational systems.`
```